src/key/mod.rs

```diff
@@ -132,10 +122,10 @@ impl AsCborValue for CoseKey {
                 return Err(CoseError::DuplicateMapKey);
             }
             seen.insert(label.clone());
-            match l {
```

Comment:
Now that `l` isn't used, can the `l.clone()` a few lines further up lose its `.clone()`?  (Similarly in header/mod.rs)

Reply:
Done